sdk/storage/azure-storage-blob/azure/storage/blob/_serialize.py

```diff
@@ -176,50 +170,41 @@ def serialize_blob_tags_header(tags=None):
     return ''.join(components)
 
 
-def serialize_blob_tags(tags=None):
-    # type: (Optional[Dict[str, str]]) -> Union[BlobTags, None]
+def serialize_blob_tags(tags: Optional[Dict[str, str]] = None) -> Union[BlobTags, None]:
```

Comment:
The return type here doesn't actually look optional? (I think the old type hint was wrong)